compressed_depth_image_transport/src/rvl_codec.cpp

```diff
@@ -1,29 +1,34 @@
+// Copyright 2012
+// All rights reserved.
+//
 // The following code is a C++ wrapper of the code presented by
 // Andrew D. Wilson in "Fast Lossless Depth Image Compression" at SIGCHI'17.
 // The original code is licensed under the MIT License.
```

Comment:
What I think we should do here is to add in the MIT license, which our linters understand; that most closely matches what this should be.  Also, the copyright year should be 2019.  Then we should add a `<license>MIT</license>` stanze to the package.xml.  And finally, we can remove the linter exclusion from CMakeLists.txt.